monocle/sanitized.py

```diff
@@ -92,6 +92,7 @@
     'REFRESH_RATE': Number,
     'REPORT_MAPS': bool,
     'REPORT_SINCE': datetime,
+    'RESCAN_UNKNOWN': int,
```

Comment:
Could you make this `Number` instead? Other numeric types are acceptable (`float`, et al.).